pkg/metrics/metrics.go

```diff
@@ -28,17 +28,18 @@ type ServerMetrics struct {
 }
 
 const (
-	metricNamespace             = "ark"
-	backupTarballSizeBytesGauge = "backup_tarball_size_bytes"
-	// TODO: Rename the Count variables to match their strings
-	backupAttemptCount           = "backup_attempt_total"
-	backupSuccessCount           = "backup_success_total"
-	backupFailureCount           = "backup_failure_total"
+	metricNamespace              = "ark"
+	backupTarballSizeBytesGauge  = "backup_tarball_size_bytes"
+	backupAttemptTotal           = "backup_attempt_total"
+	backupSuccessTotal           = "backup_success_total"
+	backupFailureTotal           = "backup_failure_total"
 	backupDurationSeconds        = "backup_duration_seconds"
 	restoreAttemptTotal          = "restore_attempt_total"
 	restoreValidationFailedTotal = "restore_validation_failed_total"
 	restoreSuccessTotal          = "restore_success_total"
 	restoreFailedTotal           = "restore_failed_total"
+	volumeSnapshotsAttemptTotal  = "volume_snapshot_attempt_total"
```

Comment:
1 more edit - it should be volumeSnapshot... (singular) instead of volumeSnapshots... (plural). Same for the next line please. Thanks!

Reply:
Fixed it, PTAL :)